protocols/purple/purple.c

```diff
@@ -1171,7 +1205,7 @@ static void handle_conv_msg(PurpleConversation *conv, const char *who, const cha
 {
 	struct im_connection *ic = purple_ic_by_pa(conv->account);
 	struct groupchat *gc = conv->ui_data;
-	char *message = g_strdup(message_);
+	char *message = purple_unescape_html(message_);
```

Comment:
Note to self: This seems unlikely to be desirable for all prpls. I think there's a prpl flag to mark it as sending html or not?